### ISSUE TITLE
libroyale-3.1.0.122 changed some function calls

### ISSUE THREAD

**Issue by pomerlef** [code-iai/pico_flexx_driver#6]
Here are a few examples of what break:
```
[...] pico_flexx_driver.cpp:245:55: error: no matching function for call to ‘royale::ICameraDevice::isCapturing()
[...] note: candidate is:
[...] virtual royale::CameraStatus royale::ICameraDevice::isCapturing(bool&)
[...] pico_flexx_driver.cpp:685:56: error: no matching function for call to ‘royale::ICameraDevice::getMaxSensorWidth()’
[...] note: candidate is:
[...] virtual royale::CameraStatus getMaxSensorWidth (uint16_t &maxSensorWidth) const = 0;
```

**Comment by kohrt** [comment]
Hi,
I don't have access to a sensor, but maybe you could change the function calls to the new API and create a pull request? @amaldo could probably test it with one of the sensors and merge the PR if everything is fine.

**Comment by nlyubova** [comment]
Hello, I've got exactly the same error. Could you please give some example how should we call it now?

**Comment by pomerlef** [comment]
For now, I rolled back to the prior version of the library. The modifications doesn't seem complicated, but I really don't have time to do the adjustments for now.

**Comment by nlyubova** [comment]
Thanks !  Managed to compile it and can do a PR when you are interested

**Comment by kohrt** [comment]
@nlyubova it would be great if you can put those changes into a PR. Thanks for the work.

**Comment by kohrt** [comment]
merged PR #7
